### PR TITLE
style: fix linting issues

### PR DESCRIPTION
**Pull request by DaniBodor**
Ruff linting and formatting is now fixed (or suppressed) and passes.

@psomhorst , could you check whether you think we need docstrings in the reader module and either add them to this PR or remove the todo I added in [609b788](https://github.com/EIT-ALIVE/eitprocessing/pull/159/commits/609b788b3869537074984beba3de9ba21cbc72c2)

### CODE REVIEW

File: eitprocessing/eit_data/event.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 @dataclass
 class Event:
+    """Single time point event registered during an EIT measurement on a Draeger machine."""
```
Comment (psomhorst) [inline eitprocessing/eit_data/event.py:6]:
```suggestion
    """Single time point event registered during an EIT measurement."""
```

File: eitprocessing/filters/butterworth_filters.py
Comment (psomhorst) [inline eitprocessing/filters/butterworth_filters.py:1]: Why did you make ignore_max_order a normal attribute? Our idea was to only have it check at initialisation, not when applying the filter. If you save the attribute, one might expect us to check the maximum order at application instead.

File: eitprocessing/eit_data/event.py (hunk 1 above)
Comment (DaniBodor) [reply eitprocessing/eit_data/event.py:6]: Currently, we are only importing it in the draeger module, which is why I put it like this. I think let's just remove the specification. We we will probably anyway ultimately settle on some unified nomenclature when we work out the `SparseData` functionality.

File: eitprocessing/plotting/__init__.py
[the remark this answers is not in the document]
Comment (DaniBodor) [reply eitprocessing/plotting/__init__.py:1]: That is indeed why I added it. I am a bit confused, did it not appear or something?

File: eitprocessing/filters/butterworth_filters.py
Comment (DaniBodor) [reply eitprocessing/filters/butterworth_filters.py:1]: Indeed, I forgot about that. I reverted this now.

**Comment by DaniBodor** [comment]
Hi @psomhorst , can you give it another look now and tell me if it's ok now.

**Pull request merged**